packages/web/docs/src/app/product-updates/(with-blog-header)/layout.tsx

```diff
@@ -1,8 +1,11 @@
-import type { ReactElement } from 'react';
```

Comment:
What's up with this `(with-blog-header)` directory?

Reply:
to group all pages from product updates and have `blog header` (name of folder group describe itself)

Reply:
It's so weird.